data/json/items/book/spiritual.json

```diff
@@ -654,5 +654,14 @@
     "weight": "292 g",
     "volume": "750 ml",
     "copy-from": "holybook_abstract"
+  },
+  {
+    "id": "holybook_asatru",
+    "type": "BOOK",
+    "name": { "str": "The Asatru Edda: Sacred Lore of the North", "str_pl": "copies of The Asatru Edda: Sacred Lore of the North" },
+    "description": "A commercial-grade, hardcover English translation of The Asatru Edda.  It serves as a modern guide to the religious and mythological beliefs of the followers of Asatru, a contemporary revival of Norse paganism.  This book contains a collection of poems, stories, rituals, and information about the gods, goddesses, heroes, and cosmology of the Norse tradition, from ancient writings and oral traditions.",
```

Comment:
There is no "The Asatru Edda" for there to be an English translation of, and there's no holy book in Asatru, despite someone apparently coming up with one and selling it on Amazon. If someone considers this book holy they're a deep minority and not representative of anything.

Reply:
If you say so, that's a valid reason to erase this item and use the Hávámál instead.